src/statsmodels/formula.jl

```diff
@@ -553,7 +557,7 @@ function coefnames(mf::ModelFrame)
         push!(term_names, expandtermnames(names))
     end
 
-    reduce(vcat, term_names)
+    isempty(term_names) ? Vector{Compat.UTF8String}() : reduce(vcat, term_names)
```

Comment:
Why not just `String` instead of `Compat.UTF8String` here? What are the possible output types here? Is it always an array of strings? In that case, it might be cleaner with just `reduce(vcat, String[], term_names)` which doesn't fail on empty input.